src/entities/RfqmQuoteEntity.ts

```diff
@@ -0,0 +1,52 @@
+import { RfqOrder } from '@0x/asset-swapper';
+import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
+
+import { RfqmFee } from '../services/rfqm_service';
+
+@Entity({ name: 'rfqm_quotes' })
+export class RfqmQuoteEntity {
+    @PrimaryColumn({ name: 'order_hash', type: 'varchar' })
+    public orderHash?: string;
+
+    @Column({ name: 'metatransaction_hash', type: 'varchar', nullable: true, unique: true })
+    public metaTransactionHash?: string;
```

Comment:
I think nullable fields can be defined but `null` on read, so we should add `| null` to nullable fields

Reply:
nice, I like that - just updated